core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -168,6 +168,8 @@
       "schema.registry.inter.instance.protocol";
   public static final String INTER_INSTANCE_PROTOCOL_CONFIG =
       "inter.instance.protocol";
+  private static final String SCHEMAREGISTRY_HEADERS_FORWARD_CONFIG =
+      "schema.registry.headers.forward";
```

Comment:
we don't need the schema.registry prefix for the config. Perhaps  something like `inter.instance.headers.whitelist`